src/main/haskell/kore/src/Kore/Attribute/SmtLemma.hs

```diff
@@ -0,0 +1,65 @@
+{-|
+Module      : Kore.Attribute.SmtLemma
+Description : Constructor symbol attribute
```

Comment:
Check the module description.

Reply:
Man, this is really annoying. Half of your comments are actually on lines that don't exist anymore, but it seems after force pushing github doesn't update properly.